app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
```

Comment:
I can see how that's a bit confusing. It's probably better to have "Mapping: None", or some other indication that it's NULL? Or are you suggesting that the "Mapping:" label shouldn't be shown at all if it's NULL?